recipes/newrelic/infrastructure/otel/debian.yml

```diff
@@ -0,0 +1,299 @@
+# Visit our schema definition for additional information on this file format.
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: nr-otel-collector
+displayName: New Relic Open Telemetry Collector
+description: New Relic install recipe for the Open Telemetry Collector
+repository: https://github.com/newrelic/opentelemetry-collector-releases
+
+
+installTargets:
+  - type: host
+    os: linux
+    platformFamily: "debian"
+  - type: host
+    os: linux
+    platform: "ubuntu"  
+
+keywords:
+  - Open Telmetry
+  - Otel
+  - Infrastructure
+  - Agent
+  - Linux
+  - Debian
+  - Jessie
```

Comment:
We did not upload the packages for Jessie and Stretch Debian versions, should we include them as metadata?

Those are supported platforms: https://github.com/newrelic/opentelemetry-collector-releases/blob/main/distributions/nr-otel-collector/upload-schema-linux-deb.yml#LL10C18-L10C18

Reply:
thought it will support all the versions which were supported by infra agent. i will update the pr according to the provided schema